README.md

```diff
@@ -23,12 +23,7 @@ import Ember from 'ember';
 import StateMixin from 'ember-state-services/mixin'
 
 export default Ember.Object.extend(StateMixin, {
-  stateName: 'emailEdit',
-  setupState: function(Factory, model) {
```

Comment:
Agreed. I'm going to just remove `setupState` in the example for now and then try and come up with some examples of overriding behavior with `_super()` calls.